tests/integration_tests/test_full_test.py

```diff
@@ -0,0 +1,190 @@
+# # # This source code is protected under the license referenced at
+# # # https://github.com/NRLMMD-GEOIPS.
+
+"""Pytest file for calling integration bash scripts."""
+
+import os
+import subprocess
+import pytest
+import sys
+
+integ_test_calls = [
+    "$geoips_repopath/tests/utils/check_code.sh all $recenter_tc_repopath",
+    "$geoips_repopath/docs/build_docs.sh $geoips_repopath $geoips_pkgname html_only",
+    "$geoips_repopath/docs/build_docs.sh $recenter_tc_repopath $recenter_tc_pkgname html_only",  # noqa
+    "$geoips_repopath/docs/build_docs.sh $data_fusion_repopath $data_fusion_pkgname html_only",  # noqa
+    "$geoips_repopath/docs/build_docs.sh $template_basic_plugin_repopath $template_basic_plugin_pkgname html_only",  # noqa
```

Comment:
As discussed, let's merge this PR, then open another branch/PR to remove full_test.sh (only use this one), and do any final cleanup on this PR.  I don't think we want to maintain both full_test.sh and test_full_test.py.

Reply:
Do you want to also get rid of base_test? It'd be easy to put both in the pytest file, and then put "marks" so they can be run separately

eg. `pytest -m integration full` or `pytest -m integration base`

Reply:
Issue created! :)